dib/list.go

```diff
@@ -0,0 +1,82 @@
+package dib
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"text/template"
+
+	"github.com/radiofrance/dib/dag"
+)
+
+const (
+	ConsoleFormat        = "console"
+	GoTemplateFileFormat = "go-template-file"
+)
+
+type FormatOpts struct {
+	Type         string
+	TemplatePath string
+}
+
+func GenerateList(graph *dag.DAG, opts FormatOpts) error {
+	imagesList := GetImagesList(graph)
+	if opts.Type == ConsoleFormat {
+		output := ""
+		for _, image := range imagesList {
+			output += fmt.Sprintf("%s: %s\n", image.ShortName, image.Hash)
```

Comment:
We might want to use a table library to have a nice output with the right column width. I suggest this one which looks simple and small: https://github.com/olekukonko/tablewriter
Allows pretty output litke this: https://github.com/olekukonko/tablewriter#output-10